main/display/display.c

```diff
@@ -714,8 +716,9 @@ void freeFont(font_t* font)
  * @param ch    The character bitmap to draw (includes the width of the char)
  * @param xOff  The x offset to draw the char at
  * @param yOff  The y offset to draw the char at
+ * @param slant The slope of the slant, with 1 being 45 degrees and `h` or 0 being 90 degrees (normal)
  */
-void drawChar(display_t* disp, paletteColor_t color, int h, const font_ch_t* ch, int16_t xOff, int16_t yOff)
+void drawCharItalic(display_t* disp, paletteColor_t color, int h, const font_ch_t* ch, int16_t xOff, int16_t yOff, int8_t slant)
```

Comment:
Also organizationally, I think there's enough text function stuff here to get broken out into it's own file.

Reply:
Yeah, `display.c` has accumulated an awful lot of non-display things. Can move some things to `font.c` or something.

Reply:
I need to impose a "no more than 1k lines" rule or something... MISRA compliance?

Reply:
Is this a mandatory change for this PR to be accepted? I _may_ be able to make spare cycles to do it